webapp/components/Editor/Editor.vue

```diff
@@ -183,30 +183,16 @@
 </template>
 
 <script>
+import defaultExtensions from './defaultExtensions.js'
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-01T05:58:57Z" title="Thursday, August 1st 2019, 7:58:57 am +02:00">Aug 1, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -183,30 +183,16 @@
 </template>
 
 <script>
+import defaultExtensions from './defaultExtensions.js'
```


Like this change